contracts/DecentAutonomousAdmin.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import {IHats} from "./interfaces/hats/full/IHats.sol";
+import {IHatsElectionEligibility} from "./interfaces/hats/full/IHatsElectionEligibility.sol";
+import {FactoryFriendly} from "@gnosis.pm/zodiac/contracts/factory/FactoryFriendly.sol";
+import {ERC165} from "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+import {IDecentAutonomousAdmin} from "./interfaces/IDecentAutonomousAdmin.sol";
+
+contract DecentAutonomousAdmin is
+    IDecentAutonomousAdmin,
+    ERC165,
+    FactoryFriendly
+{
+    // //////////////////////////////////////////////////////////////
+    //                         initializer
+    // //////////////////////////////////////////////////////////////
+    function setUp(bytes memory initializeParams) public override initializer {}
+
+    // //////////////////////////////////////////////////////////////
+    //                         Public Functions
+    // //////////////////////////////////////////////////////////////
+    function triggerStartNextTerm(TriggerStartArgs calldata args) public {
+        require(
+            args.hatsProtocol.isWearerOfHat(args.currentWearer, args.hatId),
+            "Not current wearer"
+        );
```

Comment:
To stick with modern solidity, can we use actual `error`s here?

Reply:
fixed